CheatSheets/windows-cheatsheet.md

```diff
@@ -35,6 +35,24 @@
 | `Ctrl + O` | Open |
 | `Ctrl` + `+` | to increase the size |
 | `Ctrl` + `-` | to decrease the size |
+|` Ctrl + F4`| Close the active document |
+|` Ctrl + D`| Delete the selected item and move it to the Recycle Bin|
+|` Ctrl + E`| Open Search(in most apps)|
+|` Ctrl + R`| To refresh the active window|
+|` Ctrl + Y`| Redo an action|
```

Comment:
This command already exists in the `General Shortcuts` section.

Reply:
Okay, sorry for that.
Now I'll resolve this.